sw-emulator/lib/periph/src/sha512_acc.rs

```diff
@@ -456,31 +493,61 @@ impl Sha512Accelerator {
     }
 
     /// Get the length of the hash
-    pub fn hash_len(&self) -> usize {
+    pub fn _hash_len(&self) -> usize {
```

Comment:
Did you intend to delete these? If these are only used in tests perhaps #[cfg(test)] is appropriate?